ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -3275,3 +3275,40 @@ def test_numpy_instance_invert__(
         method_flags=method_flags,
         on_device=on_device,
     )
+
+
+# getfield
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="numpy.array",
+    method_name="__getfield__",
+    dtype_and_x=helpers.dtype_and_values(
+    available_dtypes=helpers.get_dtypes("numeric"),
+    num_arrays=2,
+    ),
+)
+
+def test_numpy_instance_getfield__(
+    dtype_and_x,
+    frontend_method_data,
+    init_flags,
+    method_flags,
+    frontend,
+    on_device,
+):
+    input_dtypes, x = dtype_and_x
+    helpers.test_frontend_method(
+        init_input_dtypes=input_dtypes,
+        init_all_as_kwargs_np={
+            "object": x[0],
+        },
+        method_input_dtypes=input_dtypes,
+        method_all_as_kwargs_np={
+            "value": x[1],
```

Comment:
The test also seems to be failing due to `value` not being a kwarg accepted by the method.